README.md

```diff
@@ -3,6 +3,11 @@ Lilygo T-display S3 (ST7789) running ESPHome using patched tft_espi
 
 ![](https://github.com/landonr/lilygo-tdisplays3-esphome/blob/main/IMG_4200.jpg?raw=true)
 
+## Contributions
+- [@fisheradam](https://github.com/fisheradam) Docs
+- [@guillempages](https://github.com/guillempages) external component, sprites, touch support
```

Comment:
My contribution *is* using sprites to speed-up the drawing, but IMHO this is just a technical implementation detail. If I would read "someone implemented sprites" I would expect that they can be used, and with my commit, they can't.
IIRC @bradmck is working on that on an extra branch once he's finished with the stock tft_espi implementation
```suggestion
- [@guillempages](https://github.com/guillempages) external component, touch support
```

Reply:
can you reply with what you want it to say

Reply:
I would just remove the "sprites" part (as in the suggestion I made). Otherwise it looks good to me.

(Sorry I hadn't answered sooner; I was AFK for one week, and somehow I thought you had accepted the suggestion, based on the github emails I got).